### DISCUSSION TITLE
Auto Start-Stop

### DISCUSSION THREAD

**Discussion by m4tt075** [Mr-MIBonk/M.I.B._More-Incredible-Bash#39] [Q&A] [answered]
I've read the Wiki but couldn't find it, so I hope you don't mind me asking. Can MIB2 be used to activate the memory function for Auto Start-Stop on a Porsche with PCM4? It can apparently be coded using some Kufatec dongle, which is why I'm asking...

**Comment by m4tt075** [comment]
Answering my own question: I went through the source code and there is not option to activate the memory function for Auto Start-Stop. Not sure it is feasible to integrate this into MIB, but if so, it would be a great feature request.
@Mr-MIBonk Maybe something for the dark winter nights....  ;-)

**Comment by Mr-MIBonk** [comment, marked as answer]
This has nothing to do with M.I.B.! This is in FPA Dataset and Engine ECU...

**Reply by m4tt075** [reply]
Understood. Makes sense. OK, dongle it will be. Auto Start-Stop is driving me crazy...
